### ISSUE TITLE
Improve Block References formatting

### ISSUE THREAD

**Issue by kmanijak** [woothemes/woocommerce#48178]
**Issue Description:**
here's [Block references](https://github.com/woocommerce/woocommerce/blob/trunk/docs/building-a-woo-store/block-references.md) document that includes the list of all blocks based on block.json definition. The formatting can be improved.

**Goal:**
Improve formatting. Some ideas:
- Introduce tables for `attributes`, `supports`, including default value, etc.
- Sort/Categorize blocks
- Others...

**Comment by roykho** [comment]
Thanks for putting this up. I just want to note the formatting we currently have is following the same formatting as WP seen here https://developer.wordpress.org/block-editor/reference-guides/core-blocks/

Adding the `source` and `Sort/Categorize blocks` might still be a good improvement though.

**Comment by jamesckemp** [comment]
I agree that we could optimise the formatting here to improve scan-ability. I also propose implementing status codes. See the example image. I think this is an important document for developers.

![Image](https://github.com/user-attachments/assets/87498c19-2e6f-4419-a2d2-855759706156)

**Comment by robertdevore** [comment]
I like it @jamesckemp - maybe the "Status: " text can be removed and the colored circle could just go at the end of "Product Average Rating" line since we'll know what the colors mean by reading the status key 🤔

**Comment by AaronBowie** [comment]
👍

**Comment by jamesckemp** [comment]
@robertdevore Yep, fair request!

**Comment by kkoteen** [comment]
Hi @kmanijak - thanks for the report.

I know this has been open for a long time, and I appreciate your patience. We're currently going through some older WooCommerce issues, identifying what still needs fixing, and tidying up what's already fixed or out of scope.

I've taken a look and passed this over to the Developer Advocacy team.

What I'm seeing: The block references document still exists on trunk and is generated from block metadata, but the suggested formatting improvements are not present in the published reference.
